aliyun-ecs/src/main/java/org/jclouds/aliyun/ecs/domain/Tag.java

```diff
@@ -23,6 +23,10 @@
 @AutoValue
 public abstract class Tag {
 
+   public static final String DEFAULT_OWNER_KEY = "owner";
+   public static final String DEFAULT_OWNER_VALUE = "jclouds";
+   public static final String GROUP = "group";
```

Comment:
Do we need a key for the vSwitch tag?

Reply:
not sure we need that, as I'm adding the vswitchId tag using this code
```
      Map<String, String> tags = ComputeServiceUtils.metadataAndTagsAsValuesOfEmptyString(templateOptions);
      tags = new ImmutableMap.Builder()
              .putAll(tags)
              .put(vSwitchId, "")
              .build();
```